src/nix-env/nix-env.cc

```diff
@@ -906,53 +906,57 @@ static VersionDiff compareVersionAgainstSet(
 
 static void queryJSON(Globals & globals, std::vector<DrvInfo> & elems, bool printOutPath, bool printMeta)
 {
```

Comment:
Any reason for this extra block? It’s making everything indented further but doesn’t seem to bring much

Reply:
We need the destructor to run before the final newline is printed, because it prints the last `}`

Reply:
I guess we could just put the newline after the `queryJSON` call